.github/workflows/release-please.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+name: Release-Please
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        with:
+          token: ${{ secrets.GH_TOKEN }}
+          release-type: python
+          package-name: ark-resolver
+          pull-request-title-pattern: 'chore: release ${version}'
```

Comment:
I wonder if I configured it correctly.

Reply:
You could omit the `pull-request-title-pattern`, the default will result in something very similar, e.g. in the dsp-ingest project I have not set it and release-please will open the release PRs with a title like "chore(main): release 0.1.5".

Reply:
done